index.html

```diff
@@ -4,21 +4,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
     <title>Andrew's portfolio</title>
 </head>
 <body>
   <div class="container">
-      <div class="box1">MY Logo</div>
+      <div class="logo">
+          <div class="box1">MY Logo</div>
+          <div ><i class="material-icons">menu</i></div>
+        </div>
       <div class="box2">I'm Andrew Glade to see you!</div>
       <div class="box3">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
       <div class="box4">LET'S CONNECT</div>
       <div class="box5">
-          <div class="box5_1"><img src="Vector_1.png"></div>
-          <div class="box5_2"><img src="Linkedin_icon.png"></div>
-          <div class="box5_3"><img src="Vector_2.png"></div>
-          <div class="box5_4"><img src="Vector_3.png"></div>
-          <div class="box5_5"><img src="Vector_4.png"></div>
+          <div class="box5_1"><img alt="twitter" src="Vector_1.png"></div>
+          <div class="box5_2"><img alt="LinkedIn" src="Linkedin_icon.png"></div>
+          <div class="box5_3"><img alt="github" src="Vector_2.png"></div>
+          <div class="box5_4"><img alt="gmail" src="Vector_3.png"></div>
+          <div class="box5_5"><img alt="github" src="Vector_4.png"></div>
       </div>
```

Comment:
- [ ] Please make sure that the social media icons ("Twitter", "LinkedIn"...) are in a `ul` tag, each of them in a `li` tag. ; each icon has a link (`a` tag)